tests/integration/data/marks.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
```

Comment:
Yeah - it was quite some effort getting tests with the qutebrowser subprocesses running like this, but it definitely makes things quite easy and fun :wink: By the way, help with #999 would be very appreciated!